testdata/e2e/e2e_test.go

```diff
@@ -0,0 +1,326 @@
+package test
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"strings"
+	"testing"
+
+	cadvisor "github.com/google/cadvisor/info/v1"
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	statsapi "k8s.io/kubernetes/pkg/kubelet/apis/stats/v1alpha1"
+)
+
+func TestMetricSample(t *testing.T) {
+
+	wd := os.Getenv("WORKING_DIR")
+
+	parsedK8sLists := &ParsedK8sLists{
+		NodeSummaries:          make(map[string]statsapi.Summary),
+		BaselineNodeSummaries:  make(map[string]statsapi.Summary),
+		NodeContainers:         make(map[string]map[string]cadvisor.ContainerInfo),
+		BaselineNodeContainers: make(map[string]map[string]cadvisor.ContainerInfo),
+	}
+
+	t.Run("ensure that a metrics sample has expected files", func(t *testing.T) {
```

Comment:
it will fail on the first file that it does not find and raise the file name as an error. The assumption being that if one file is missing from the file set then it is incomplete and requires further investigation.